pages/search-beta.tsx

```diff
@@ -10,25 +10,33 @@ import {
   InstantSearch,
   InstantSearchServerState,
   InstantSearchSSRProvider,
+  ClearRefinements,
 } from "react-instantsearch-hooks-web"
 import { FormattedMessage, IntlProvider } from "react-intl"
 import styled from "styled-components"
 
-import {
-  SearchBox,
-  Hits,
-  Hit,
-  RefinementList,
-  PlainRefinementList,
-  Pagination,
-
-} from "components/ImprovedSearch"
+import { SearchBox, Hits, Hit, PlainRefinementList, Pagination } from "components/ImprovedSearch"
 import { Page } from "components/Page"
 import { SearchStatistics } from "components/SearchStatistics"
 import { Section } from "components/Section"
 import { searchClient, searchClientServer } from "config"
 import englishTexts from "locales/english.json"
 
+const StyledName = styled(ClearRefinements)`
```

Comment:
I think this code style is pretty "off" with our code base since you need to override styles on Algolia's CSS class names. I think we should avoid this. You should rather [use the hooks approach](https://www.algolia.com/doc/api-reference/widgets/clear-refinements/react-hooks/#hook) and use an EDS button (or whatever Espen suggests). With this approach, we can control the UI with styled-components and we avoid relying on internal Algolia class names 👍  This is also the way we do it with the rest of the components